pkg/oauth/server.go

```diff
@@ -261,6 +260,7 @@ func (s *server) RegisterRoutes(server *web.Server) {
 	group.GET("/*", webui.Template.Handler, middleware.CSRF())
 
 	// No CSRF here:
+	group.POST("/api/auth/logout", s.Logout, s.requireLogin)
```

Comment:
I don't think it's a good idea to remove CSRF protection for this route

Reply:
Ok then I see two ways to do it:
- Add the OP logout request to the backend logout handler of the console and using only that endpoint in the browser
- Implement a redirect/callback approach, similar to the one described in the article

Which do you prefer?

Reply:
I'd go for option 2 then. With 1 the Console would need to know about the session ID that the user has on the OAuth server, and that would be a bit smelly.

Reply:
Ok, can someone help me on the backend side with this? I already have a lot of stuff to do on the frontend as far as the Account app goes. cc @bafonins

Reply:
will try to look into this, however I also have quite some tasks for this and next milestones